File: src/main/java/com/lannstark/lec06/Lec06Main.java
package com.lannstark.lec06;

import java.util.Arrays;
import java.util.List;

public class Lec06Main {

    public static void main(String[] args) {

        // 1.
        List<Long> numbers = Arrays.asList(1L, 2L, 3L);
        for (long number : numbers) {
            System.out.println(number);
        }

        // 2.
        for(int i = 1; i <= 3; i++) {
            System.out.println(i);
        }
        for(int i = 3; i >= 1; i--) {
            System.out.println(i);
        }

        // 3.
        for (int i = 1; i <= 5; i += 2) {
            System.out.println(i);
        }

        int i = 1;
        while (i <= 3) {
            System.out.println(i);
            i++;
        }

        // 4.
        int i1 = 1;
        while(i1 <= 3) {
            System.out.println(i1);
            i1++;
        }

    }
}

File: src/main/java/com/javaToKotlin/lec06/Lec06Main.kt
package com.javaToKotlin.lec06

fun main() {
    // 1.
    val numbers = listOf(1L, 2L, 3L)
    for (number in numbers) {
        println(number)
    }

    // 2.
    for (i in 1..3) {
        println(i)
    }
    for (i in 3 downTo 1) {
        println(i)
    }

    // 3.
    for (i in 1..5 step 2) {
        println(i);
    }

    // 4.
    var i1 = 1
    while(i1 <= 3) {
        println(i1)
        i1++
    }
}